src/main/java/org/example/services/JobRoleService.java

```diff
@@ -52,4 +54,18 @@ public JobRoleDetailedResponse getJobRoleById(final int id)
     public int getTotalRecords() throws SQLException {
         return jobRoleDao.getTotalOpenJobs();
     }
-}
+    public int createJobRole(
+            final JobRoleRequest jobRoleRequest)
+            throws FailedToCreateException, SQLException,
+            DoesNotExistException {
```

Comment:
I cant see anywhere in the code that a DoesNotExistException is thrown in this call stack.  I think you can remove it...

Reply:
Removed